src/app/api/patient/auth/signUp/route.ts

```diff
@@ -0,0 +1,43 @@
+import { getUserByEmail } from "@server/mongodb/actions/User";
+import APIWrapper from "@server/utils/APIWrapper";
+import User from "@server/mongodb/models/User";
+
+export const POST = APIWrapper({
+  config: {
+    requireToken: true,
+  },
+  handler: async (req) => {
+    const { email, name, phoneNumber, patientDetails } = await req.json();
```

Comment:
This route should still accept birthdate, secondaryContactName, and secondaryContactPhone separately instead of as patientDetails